backend/src/db/migrations/20230917152623_add_unknown_genre.ts

```diff
@@ -0,0 +1,23 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'genres';
+
+const GENRE_KEY_COLUMN_NAME = 'key';
+
+const UNKNOWN_GENRE = { key: 'unknown', name: 'Unknown' } as const;
+
+const up = async (knex: Knex): Promise<void> => {
+  return await knex(TABLE_NAME)
+    .where(GENRE_KEY_COLUMN_NAME, UNKNOWN_GENRE.key)
+    .then(async (rows) => {
+      if (!rows.length) {
+        await knex(TABLE_NAME).insert(UNKNOWN_GENRE);
+      }
+    });
+};
+
+const down = (knex: Knex): Promise<void> => {
+  return knex(TABLE_NAME).where({ key: UNKNOWN_GENRE.key }).del();
```

Comment:
If we have condition to insert `unknown` genre (when it does not exists), then we should not remove it from table on rollback